packages/interbit/src/file/packCovenants.js

```diff
@@ -6,20 +6,33 @@ const {
   constants: { ROOT_CHAIN_ALIAS }
 } = require('interbit-covenant-tools')
 
+const log = require('../log')
 const { startInterbit } = require('../chainManagement')
 
 // Note: Because interbit-core does not expose it's hashing function (yet) we ned to
 // start a cli and deploy the covenants to get a valid hash that is in sync with interbit
 // TODO: update this to use the hashing function instead of starting a CLI -- Blocked until core releases hashing algo in API
+/**
+ * Packs the covenants specified in covenantConfig and stores them in the artifacts
```

Comment:
npm pack - all covenants are npm packages and thus can be packed then have the binary hashed. If we don't have a ticket for covenant writing tutorial we should make one. I think it would lend a lot of clarity to the variable named